src/frontend/komponenter/Fagsak/Behandlingsresultat/EndretUtbetalingAndelRad.tsx

```diff
@@ -107,68 +106,43 @@ const EndretUtbetalingAndelRad: React.FunctionComponent<IEndretUtbetalingAndelRa
                                     : Status.ADVARSEL
                             }
                         />
-                        {endretUtbetalingAndel.personIdent
-                            ? lagPersonLabel(
-                                  endretUtbetalingAndel.personIdent,
-                                  åpenBehandling.personer
-                              )
-                            : 'Ikke satt'}
+                        <BodyShort size={'small'}>
```

Comment:
Å legger til Bodyshort med størrelse small er egentlig en fix på problemet som blir løst i Aksel v5. Skal jeg la det være sånn her eller skal jeg prøve å legge på className="tabell"?

Reply:
Sett på tabell, så blir det lettere å se hva som skal fjernes og oppgraderes i neste runde 😌

Reply:
🙌 fikser!

Reply:
Får det ikke til å funke ordentlig, pga bruken av expandable row her. Blir rar styling 😞 jeg kan legge inn en kommentar på favro om at BodyShort her er brukt for å kompensere for stylingen av tabellen
![image](https://github.com/navikt/familie-ba-sak-frontend/assets/25459913/9c757d3b-5c9a-4642-8157-20725a14fb07)

Reply:
Ja, gjør det! 🙂

Reply:
Tenker at et lite løft er også et viktig løft. Vil heller ha inn dette og være et stort steg nærmere å være helt oppgradert, enn å la det ligge mens du er på ferie